addons/image/iio.c

```diff
@@ -87,6 +87,7 @@ bool al_init_image_addon(void)
          success |= al_register_bitmap_loader_f(extensions[i], _al_load_android_bitmap_f);
          //success |= al_register_bitmap_saver(extensions[i], _al_save_android_bitmap);
       }
+      success |= al_register_bitmap_identifier(".webp", _al_identify_webp);
```

Comment:
Why is this done? (inside of ALLEGRO_CFG_IIO_HAVE_ANDROID)

Reply:
Android supports WebP with system facilities and _al_identify_webp doesn't need libwebp, so I've added it there as well to make WebP identification work on Android too.

Reply:
Don't you still have to register .webp somehow for al_load_bitmap to know to use those system facilities? I don't doubt it works, I'm just curious how exactly...

Reply:
Look at the loop, a few lines above. It was already there :)

Reply:
Oh, interesting, I did not know that.